fe/fe-core/src/main/java/com/starrocks/ha/BDBHA.java

```diff
@@ -106,6 +110,19 @@ public boolean fencing() {
         return false;
     }
 
+    private long getLargestEpoch(Database epochDB) {
```

Comment:
I think `getLatestEpoch` is a better name. I suggest you add an asert to ensure that the latest epoch >= db.count().